mlir/test/mlir-miopen-driver/layout_driver.mlir

```diff
@@ -0,0 +1,7 @@
+// Check the guards of tensor layouts in miopen-driver 
+
+// XFAIL:* 
```

Comment:
There are 4 failing commands but only 1 XFAIL. 
please try concatenate 4 mlir-miopen-driver invocations into 1 RUN, NAND the return values of each invocation.

In such case , should any of them have return value of 0, the overall result would become 1 and fail the test